types/reefjs/index.d.ts

```diff
@@ -1,173 +1,84 @@
-// Type definitions for reefjs 7.6
+// Type definitions for reefjs 12.1
 // Project: https://github.com/cferdinandi/reef#readme
-// Definitions by: shockdevv <https://github.com/shockdevv>
+// Definitions by: bryandonmarc <https://github.com/bryandonmarc>
+//                 shockdevv <https://github.com/shockdevv>
 //                 cferdinandi <https://github.com/cferdinandi>
```

Comment:
Please remove @cferdinandi from the list since he's the original author, not the types author.